packages/devtools/lib/src/timeline/frames_bar_plotly.dart

```diff
@@ -59,39 +67,85 @@ class FramesBarPlotly {
     );
   }
 
+  // Return a list of all of traces in trace index order:
+  // e.g., [
+  //         GPU Good Trace Data,   // array index gpuGoodTraceIndex
+  //         GPU Jank Trace Data,   // array index gpuJankTraceIndex
+  //         GPU Select Trace Data, // array index gpuSelectTraceIndex
+  //         CPU Good Trace Data,   // array index cpuGoodTraceIndex
+  //         CPU Jank Trace Data,   // array index cpuJankTraceIndex
+  //         CPU Select Trace Data, // array index cpuSelectTraceIndex
+  //       ]
   static List<Data> createFPSTraces() {
+    final List<Data> allTraces = [];
+
     // Strange plotly bug with initial setup of x,y.  If x and y are empty array
     // then the first entry, for each trace, isn't rendered but hover does
     // display the Y value.  So prime each trace with some data.  Added
     // at x-axis coord of xCoordNotUsed (-1) (hide rangemode: nonnegative
     // displays at 0 and greater) and y is zero.
-    final Data traceCpuGood = Data(
+
+    // trace GPU Good
+    allTraces.insert(gpuGoodTraceIndex, Data(
       y: [yCoordNotUsed],
       x: [xCoordNotUsed],
       type: 'bar',
       legendgroup: 'good_group',
-      name: 'CPU',
+      name: 'GPU',
       hoverinfo: 'y+name',
       marker: Marker(
-        color: colorToCss(mainCpuColor),
+        color: colorToCss(mainGpuColor),
       ),
       width: [0],
-    );
+    ));
```

Comment:
nit: can we add a trailing comma after the Data() object for all these `allTraces.insert(...)` calls so that we are using fluter style?

Reply:
Done.